server/game/cards/08-MotC/LetterFromTheDaimyo.js

```diff
@@ -0,0 +1,29 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class LetterFromTheDaimyo extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title: 'Make opponent discard 2 cards',
+            cost: AbilityDsl.costs.sacrificeSelf(),
+            when: {
+                afterConflict: (event, context) => context.source.parent.isParticipating() &&
+                                                   event.conflict.winner === context.source.parent.controller &&
+                                                   event.conflict.conflictType === 'political'
+            },
+            gameAction: AbilityDsl.actions.chosenDiscard(context => ({ target: context.player.opponent, amount: 2 }))
```

Comment:
`gameAction: AbilityDsl.actions.chosenDiscard({ amount: 2 })` is fine here because the opponent is the default target